streaming/streaming.go

```diff
@@ -151,8 +151,7 @@ func (c *Conn) manageForever() {
 
 	for {
 		if err := c.connect(); err != nil {
-			log.Printf("luno/streaming: Connection error key=%s pair=%s: %v",
-				c.keyID, c.pair, err)
+			log.Printf("luno/streaming: Connection error: %v", err)
```

Comment:
log/slog is available (maybe not for the version we support for luno-go so you can log parameters.

Reply:
I think using slog will defeat the purpose of this change.

Reply:
Is the purpose only be able to collect all logs through searching "luno/streaming: Connection error:" on something like Kibana or is it to consolidate metrics?

Reply:
Was going to say we can keep pair, but don't know if it's that useful for this error? 🤔

Reply:
The purpose is to collate the logs. The pair and key is not particularly useful.